src/pages/workspaces/workspace/Workflows.js

```diff
@@ -0,0 +1,163 @@
+import _ from 'lodash/fp'
+import { div, h, span } from 'react-hyperscript-helpers'
+import { AutoSizer } from 'react-virtualized'
+import * as breadcrumbs from 'src/components/breadcrumbs'
+import { centeredSpinner, icon } from 'src/components/icons'
+import { FlexTable, TextCell } from 'src/components/table'
+import { Rawls } from 'src/libs/ajax'
+import { reportError } from 'src/libs/error'
+import * as Nav from 'src/libs/nav'
+import * as Style from 'src/libs/style'
+import * as Utils from 'src/libs/utils'
+import { Component } from 'src/libs/wrapped-components'
+import WorkspaceContainer from 'src/pages/workspaces/workspace/WorkspaceContainer'
+
+
+const styles = {
+  pageContainer: {
+    margin: '1rem', height: 500, display: 'flex'
+  },
+  submissionsTable: {
+    flex: '1 1 auto'
```

Comment:
nit: this would usually be better as `flex: 1 1 0px`, which is equivalent to the shorthand `flex: 1`. It probably doesn't actually matter here, but in situations where you're trying to create proportional layouts, an `auto` basis will throw things off by taking into account the content size, while `0px` will ignore it.